.ci/jenkins/Jenkinsfile.promote

```diff
@@ -18,8 +18,8 @@ pipeline {
         KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
 
         // Linked to node label
-        CONTAINER_ENGINE='podman'
-        CONTAINER_ENGINE_TLS_OPTIONS='--tls-verify=false'
+        CONTAINER_ENGINE = 'podman'
```

Comment:
Why we have podman in the promote and docker in the deploy ?

Reply:
in deploy, we need `docker` for the multiplatform build
in promote, we need the a container engine only for login to registry, so `podman` should be the default